packages/react-components/react-nav-preview/src/components/sharedNavStyles.styles.ts

```diff
@@ -0,0 +1,82 @@
+import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { makeResetStyles, makeStyles, shorthands } from '@griffel/react';
+
+// Styles shared by several nav components.
+
+export const navItemTokens = {
+  indicatorOffset: 18,
+  indicatorWidth: 4,
+  indicatorHeight: 20,
+};
+
+/**
+ * Styles for the root slot
+ */
+export const useRootDefaultClassName = makeResetStyles({
```

Comment:
Would be good to mention which components will share this style.
```suggestion
 * Styles for the root slot used in Nav components such as NavCategory and NavItem.
 */
export const useNavRootDefaultClassName = makeResetStyles({
```